src/components/MessageList/TypingIndicatorContainer.js

```diff
@@ -1,10 +1,10 @@
 import React, { useContext } from 'react';
-import styled from '@stream-io/styled-components';
+import styled from 'styled-components/native';
 
 import { ChannelContext, ChatContext } from '../../context';
 
 const Container = styled.View`
-  bottom: 0;
+  bottom: 0px;
```

Comment:
we just gonna wait to remove these unneeded values later?

Reply:
Yeah I didn't touch the js files again after I got the ts of styled components happy, we need to just hit them well as we convert